### PR TITLE
Move deflectQml to deflect/qt, rename library to DeflectQt

### PR DESCRIPTION
**Pull request by dnachbaur**

### CODE REVIEW

File: deflect/qt/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Copyright (c) 2015, EPFL/Blue Brain Project
+#                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
+
+set(DEFLECTQT_MOC_HEADERS EventReceiver.h QmlStreamerImpl.h)
+set(DEFLECTQT_PUBLIC_HEADERS QmlStreamer.h)
+set(DEFLECTQT_SOURCES EventReceiver.cpp QmlStreamer.cpp QmlStreamerImpl.cpp)
+set(DEFLECTQT_LINK_LIBRARIES
+  PUBLIC Deflect Qt5::Quick PRIVATE Qt5::Qml)
+set(DEFLECTQT_INCLUDE_NAME deflect/qt)
+set(DEFLECTQT_NAMESPACE deflectqt)
```
Comment (rdumusc) [inline deflect/qt/CMakeLists.txt:11]: is this correct..?

**Comment by rdumusc** [comment]
+1 otherwise

File: deflect/qt/CMakeLists.txt (hunk 1 above)
Comment (dnachbaur) [inline deflect/qt/CMakeLists.txt:11]: Unfortunately yes. It is the concatenated namespaces that form the DEFLECTQT_API declaration in this case (for the dll export stuff). If unspecified, it uses PROJECTNAME, which would not be correct as then the macro would declare 'import'. It should be cleaned up in CMake/common; it has almost nothing to do with namespaces, more like the library/target name now.

Comment (rdumusc) [inline deflect/qt/CMakeLists.txt:11]: OK, yes, should be cleaned in CMake/common soon, this is confusing!

**Pull request merged**